### ISSUE TITLE
Not a working version

### ISSUE THREAD

**Issue by dfrumkin** [rmalav15/Super-SloMo#5]
While the code is educational and has certain value, it would be nice to put a note in the readme that this is not a working version of SuperSlomo, but rather a research / work in progress / ...

For example,

1. Even the GIF that you put in the README does not do SloMo.  I see frames almost duplicated.  This is also what happens with the pretrained model on my own examples.
2. Your UNet has much lower capacity than the one described in the paper.  The layers are not duplicated; the 7x7 and 5x5 kernels are only in one of the UNets; also, looks like you played with final activations a bit (probably a good idea, but not what is written in the paper).  Just look at the paper or the public pytorch implementation for comparison.

**Comment by rmalav15** [comment]
Hi @dfrumkin 

1. Thank you for looking into this. Someone else also pointed it out earlier. I have added the note.
2. I took some liberty in playing with model and its hyper-parameters to suit my hardware capabilities. I will try to further tune the model and strictly follow model guidelines from paper to provide a infallible (for the current standards) model.
 
Much Thanks.
Ram

**Comment by jamesdh** [comment]
I just went through the hassle of checking this out and updating it to work with tensorflow 2 on Apple Silicon, only to discover that it's not doing interpolation at all. Please update the readme with a more descriptive caveat to save others the loss of time. Even without interpolation, this barely runs faster than [PyTorch Super-SloMo](https://github.com/avinashpaliwal/Super-SloMo) running entirely on the CPU, and the end result is equivalent to a one-liner on the CLI that will give you the same thing almost instantly. 

`ffmpeg -i dog.mp4 -filter:v "setpts=2.0*PTS" dog_slomo.mp4`

Then running the following would give you something closer to the result that was actually intended by the original SuperSloMo: 

`ffmpeg -i dog_slomo.mp4 -filter:v minterpolate -r 120 dog_slomo_smooth.mp4`
